js/app/dashboard/components/context-menus/TasksContextMenu.js

```diff
@@ -184,9 +188,13 @@ const DynamicMenu = () => {
   const couriers = useSelector(selectCouriersWithExclude)
   const tasksListsLoading = useSelector(selectTaskListsLoading)
 
```

Comment:
and then used as 

`const assign = useAssignAction()`

...

`  const assignSelectedOrders = (username) => assign(username, selectedOrders)`